test/train-sets/ref/automl_readable_cubic.txt

```diff
@@ -254,7 +254,7 @@ weighted_labeled_examples 3
 weighted_labels 0
 weighted_unlabeled_examples 0
 example_number 3
-total_features 508674
+total_features 506754
```

Comment:
this stat changes because we are not learning on the champ at the end, so the stats reflect the last config being learned

Reply:
https://github.com/VowpalWabbit/vowpal_wabbit/blob/2639495b373c8d334b74d2a25851400271731c2b/vowpalwabbit/core/include/vw/core/example.h#L122

its related to all that state

Reply:
just updated to learn after to fix feature count